content/en/docs/api-documentation/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "KubeSphere API"
+description: "How to use KubeSphere API to build your own application"
+layout: "single"
+
+linkTitle: "API Documentation"
+
+weight: 3100
+
+icon: "/images/docs/docs.svg"
+
+---
+
+## [API Documentation](./kubesphere-api/)
+
+The REST API is the fundamental fabric of KubeSphere. This page show you how to access KubeSphere API server.
```

Comment:
show => shows
how to access KubeSphere API server. => how to access the KubeSphere API server.